include/caf/ble_smp.rst

```diff
@@ -22,6 +22,11 @@ To use the module, you must enable the following Kconfig options:
 * :option:`CONFIG_BOOTLOADER_MCUBOOT` - This option enables the MCUboot bootloader.
   The DFU over Simple Management Protocol in Zephyr is supported only with the MCUboot bootloader.
 
+Optionally, you can also enable :option:`CONFIG_MCUMGR_CMD_OS_MGMT`.
+This option enables MCUmgr OS management handlers, which can be used to remotely trigger device reboot after image transfer is completed.
+After the reboot, device can start using the new firmware.
+The remote reboot functionality is used for example by `nRF Connect for Mobile`_ application.
+
```

Comment:
@greg-fer in sensor_sampler we made additional paragraph for Kconfig options that enable additional features. Shall we do the same in this case? Something like:

```
Enabling remote OS management
=============================
```

Reply:
Yes.
```
Enabling remote OS management
=============================

The |smp| supports remote OS management feature, which you can enable using the following Kconfig option:

* :option:`CONFIG_MCUMGR_CMD_OS_MGMT` - This option enables MCUmgr OS management handlers.
  Use these handlers to remotely trigger the device reboot after the image transfer is completed.
  After the reboot, the device starts using the new firmware.
  One of the applications that support the remote reboot functionality is `nRF Connect for Mobile`_.
```

Reply:
Ok, changed